dlrs/main/lwc/rollupEditor/rollupEditor.html

```diff
@@ -1,269 +1,542 @@
+                        data-name="rollup_CalculationMode__c"
+                        label="Calculation Mode"
+                        placeholder="Select"
+                        options={calculationModes}
+                        value={rollup.CalculationMode__c}
+                        required
+                      ></lightning-combobox>
+                      <c-rollup-editor-error
+                        errors={errors.FieldToOrderBy__c}
+                      ></c-rollup-editor-error>
+                  </div>
+
+                  <!-- Calculation Sharing Mode -->
+                  <div class="slds-m-top_x-small">
+                      <lightning-combobox
+                        name="rollup_CalculationSharingMode__c"
+                        data-name="rollup_CalculationSharingMode__c"
+                        label="Calculation Sharing Mode"
+                        placeholder="Select"
+                        options={calculationSharingModes}
+                        value={rollup.CalculationSharingMode__c}
+                        required
+                      ></lightning-combobox>
+                      <c-rollup-editor-error
+                        errors={errors.FieldToOrderBy__c}
+                      ></c-rollup-editor-error>
+                  </div>
+
+                </div>
+
+            </div>
+          </div>
+        </div>
+
+        <!-- Text Lookups Section -->
+        <div class="slds-m-top_large">
+          <div class="slds-section" data-section-name="section_text-lookups">
+            <h3 class="slds-section__title">
+              <button onclick={onToggleSection} data-section-button="section_text-lookups" class="slds-button slds-section__title-action">
+                <svg class="slds-section__title-action-icon slds-button__icon slds-button__icon_left" >
+                  <use xlink:href="/_slds/icons/utility-sprite/svg/symbols.svg#switch"></use>
+                </svg>
+                <span class="slds-truncate" title="Text Lookups">Text Lookups</span>
+              </button>
+            </h3>
+            <div  class="slds-section__content" >
+
+              <div class="slds-grid slds-gutters slds-wrap">
+                <!-- Concatenate Delimiter -->
+                <div class="slds-col slds-size_1-of-1 slds-medium-size_1-of-2 slds-m-top_x-small">
+                    <lightning-input
+                      type="text"
+                      name="rollup_ConcatenateDelimiter__c"
+                      data-name="rollup_ConcatenateDelimiter__c"
+                      label="Concatenate Delimiter"
+                      value={rollup.ConcatenateDelimiter__c}
+                    ></lightning-input>
+                    <c-rollup-editor-error
+                      errors={errors.ConcatenateDelimiter__c}
+                    ></c-rollup-editor-error>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <!-- Description Section -->
```

Comment:
I never loved the Description so far down in the other editor and having a distinct collapsed section feels unnecessary.

What do you think about moving it up into the "Information" section?